src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -830,33 +816,29 @@ public List<EquipmentModificationInfos> createLoadModification(UUID networkUuid,
     }
 
     private List<EquipmentModificationInfos> execCreateGeneratorModification(NetworkStoreListener listener,
-                                                                             GeneratorModificationInfos generatorModificationInfos,
-                                                                             UUID repordId, String reporterId) {
+                                                             GeneratorModificationInfos generatorModificationInfos,
+                                                             UUID repordId, String reporterId) {
         Network network = listener.getNetwork();
         String rootReporterId = reporterId + "@" + NETWORK_MODIFICATION_TYPE_REPORT;
         ReporterModel reporter = new ReporterModel(rootReporterId, rootReporterId);
         String subReportId = "Generator modification " + generatorModificationInfos.getEquipmentId();
         Reporter subReporter = reporter.createSubReporter(subReportId, subReportId);
 
         return doAction(listener, () -> {
-            if (listener.isApplyModifications()) {
-                try {
-                    Generator generator = getGenerator(network, generatorModificationInfos.getEquipmentId());
-                    // modify the generator in the network
-                    modifyGenerator(generator, generatorModificationInfos, subReporter);
-                } catch (NetworkModificationException exc) {
-                    subReporter.report(Report.builder()
-                        .withKey("generatorModification")
-                        .withDefaultMessage(exc.getMessage())
-                        .withValue("id", generatorModificationInfos.getEquipmentId())
-                        .withSeverity(TypedValue.ERROR_SEVERITY)
-                        .build());
-                }
-            }
 
-            // add the generator modification entity to the listener
-            listener.storeGeneratorModification(generatorModificationInfos);
-        }, MODIFY_GENERATOR_ERROR, repordId, reporter, subReporter).stream().map(EquipmentModificationInfos.class::cast)
+            try {
+                Generator generator = getGenerator(network, generatorModificationInfos.getEquipmentId());
+                // modify the generator in the network
+                modifyGenerator(generator, generatorModificationInfos, subReporter);
+            } catch (NetworkModificationException exc) {
```

Comment:
Remove it

Reply:
why ? I suggest that if it's for a refacto, we process it in another US